test/e2e/performanceprofile/functests/utils/cgroup/v2/v2.go

```diff
@@ -98,3 +92,24 @@ func (cm *ControllersManager) Child(ctx context.Context, pod *corev1.Pod, contai
 	}
 	return nil
 }
+func (cm *ControllersManager) execAndStore(pod *corev1.Pod, containerName, dirPath string, store map[string]*string) error {
```

Comment:
this seems one of the instances on which seems better to generalize the function. But I think you did not because the v1 and v2 package have no common package to share library functions, and adding one doesn't seem so great. If that's the case I agree to keep it like this for starters.

Reply:
Thanks. We may consider shared library in the future in case of additional common functions.